openapi.yaml

```diff
@@ -50,17 +97,51 @@ info:
   version: '0.1'
 tags:
 - name: action
-  description: |
-    ### CKAN's Action API is a powerful, RPC-style API that exposes all of CKAN's core features
+  description: 
+    en: |
+      ### CKAN's Action API is a powerful, RPC-style API that exposes all of CKAN's core features
+
+      All endpoints support POST requests with an `application/json` or a
+        `multipart/form-data` body.
+    -   Use an `application/json` POST request when nested json parameters
+        are required
+    -   Use a `multipart/form-data` POST request when file uploads are required
 
-    All endpoints support POST requests with an `application/json` or a
-      `multipart/form-data` body.
-    - Use an `application/json` POST request when nested json parameters
-      are required
-    - Use a `multipart/form-data` POST request when file uploads are required
+      Read-only endpoints (`*_show`, `*_list`, `*_search` etc.)
+      accept GET requests with query parameters.
+    fr: |
+      ### L’API d’action de CKAN est une puissante API de type appel de procédure à distance qui expose toutes les fonctionnalités de base de CKAN. 
+ 
+      Tous les points terminaux prennent en charge les demandes POST avec un corps « application/json » ou un corps 
+        « multipart/form-data ». 
+-       Utiliser une demande POST « application/json » lorsque des paramètres json imbriqués sont nécessaires. 
+-       Utiliser une demande POST « multipart/form-data » lorsque des téléchargements de fichiers sont nécessaires. 
+ 
+      Points terminaux en lecture seule (« *_montrer », « *_liste », « *_recherche », etc.) 
+      Accepter les demandes d’obtention de données (GET) avec des paramètres de requête. 
+ 
+Les réponses sont renvoyées dans une enveloppe JSON en cas de succès : 
+« `json 
+{ 
+  “aide” : “url pour compléter l’aide sur les points terminaux”, 
+  “succès” : vrai, 
+  “résultat” : {…} 
```

Comment:
the keys in these examples can't be translated, and the quotes need to be ascii `"` quotes

Reply:
Quotes are now ascii quotes, keys have been updated